service/src/main/java/bio/terra/cbas/controllers/MethodsApiController.java

```diff
@@ -46,7 +46,7 @@
 @Controller
 public class MethodsApiController implements MethodsApi {
   private final CromwellService cromwellService;
-  private final MethodDao methodDao;
+  private static MethodDao methodDao;
```

Comment:
This looks super suspicious to me, especially since you're still supplying a `methodDao` and overwriting the value in the constructor. What was the motivation behind this, and is it possible to revert it back to `final` and not `static`?

Reply:
I wasn't sure what the best way to handle the error that I was getting. When this is `final` line 245 throws an error ```Non-static field 'methodDao' cannot be referenced from a static context```. Making the function non-static seemed like not the right thing to do either, so I'd be curious what the right thing to do here is in this situation.

Reply:
In this situation, the other method _should_ be non-static, because it's now relying on a class field (methodDao) that gets supplied at construction time. So it should be fine to make that other method an instance method (ie non-static)

This actually neatly dovetails with my other comment (about time of check / time of use race conditions). The reason the other validate method is no longer static is because it's now checking live database state, instead of inherent properties of the method.